src/components/notification/RegularNotification.jsx

```diff
@@ -55,33 +56,83 @@ const styles = StyleSheet.create({
   user: {
     fontWeight: 'bold',
   },
+  buttonContainer: {
+    flex: 1,
+    height: 70,
+    width: 75,
+    backgroundColor: 'rgba(255,255,255,1)',
```

Comment:
add colors to the color.js file and import it instead of creating your own

Reply:
we can do this later. not high prio